### PR TITLE
[Fix] File unhighlightning

### PR DESCRIPTION
**Pull request by JulesGuesnon**
Hey!
While using the application it was kinda frustrating that you can't remove the highlight of a file once you clicked it. As it's the behavior of the native explorers, I implemented it

https://user-images.githubusercontent.com/36401048/217800481-325341ca-8895-4853-8e29-ccdf909e7bd9.mov

### CODE REVIEW

**Review by jamiepine — DISMISSED** [review]
Great addition, the existing way was to click the file again, which was temporary until I or someone else added this, so thank you!

File: packages/interface/src/components/explorer/VirtualizedList.tsx [hunk 1]
```diff
@@ -117,7 +117,20 @@ export const VirtualizedList = memo(({ data, context, onScroll }: Props) => {
...
+			<div
+				ref={scrollRef}
+				className="h-screen custom-scroll explorer-scroll"
+				onClick={(e) => {
+					if (
+						!scrollRef.current ||
+						(!(e.target as HTMLElement).classList.contains('file-row') &&
+							scrollRef.current !== e.target)
+					)
+						return;
```
Comment (Brendonovich) [inline packages/interface/src/components/explorer/VirtualizedList.tsx:129]: Could this check be avoided if the virtualizer's children had `stopPropagation` in their `onClick` handlers?

Comment (JulesGuesnon) [reply packages/interface/src/components/explorer/VirtualizedList.tsx:129]: I actually tried and it wasn't working, not sure why tho. Let me double check

Comment (JulesGuesnon) [reply packages/interface/src/components/explorer/VirtualizedList.tsx]: Ok I don't get why it wasn't working the first time, but it is now, so definitely shorter and more efficient to do it with `stopPropgation`

**Review by Brendonovich — APPROVED** [review]

**Pull request merged**